Gemfile.lock

```diff
@@ -0,0 +1,53 @@
+GEM
```

Comment:
I don't think this file should be added. `Gemfile` already has the dependency list.

Reply:
Hey, thank you again for reviewing!

In Ruby land it's very standard to check-in the Gemfile.lock.  And in fact the bundler documentation [suggests it](http://bundler.io/v1.15/guides/bundler_sharing.html).

The reason is that `Gemfile.lock` captures the exact versions of the specified gems and of the dependent gems that are pulled-in transitively.  It allows anyone checking-out the repository to build with the same versions of every gem, exactly matching those in use by the person who checked-in or pushed.

However I won't die on this hill! If you want it removed to approve, I'll make it so.

Reply:
Interesting, thanks for the link.

Reply:
Thank you for looking at the PR's!